.vscode/settings.json

```diff
@@ -3,10 +3,15 @@
         120
     ],
     "editor.formatOnSave": true,
-    "python.formatting.provider": "black",
-    "python.linting.enabled": true,
-    "python.linting.flake8Enabled": true,
-    "python.linting.flake8Args": [
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnSave": true
+    },
+    "black-formatter.args": [
+        "--line-length",
+        "120"
+    ],
+    "flake8.args": [
```

Comment:
Does flake8 need to enabled independent of flake8 args?

Reply:
That's right -- flake8 needs to be installed as an extension rather than enabled via `settings.json` (same as black). From [this documentation](https://marketplace.visualstudio.com/items?itemName=ms-python.flake8):

```
Once installed in Visual Studio Code, flake8 will be automatically executed when you open a Python file.

If you want to disable flake8, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) per workspace in Visual Studio Code.
```

Reply:
Gotcha!